res/controllers/common-hid-packet-parser.js

```diff
@@ -1658,6 +1637,20 @@ class HIDController {
                     for (const bit_id in field.value.bits) {
                         const bit = field.value.bits[bit_id];
                         bit.packet = packet;
+                        // Fill lookup map
+                        if (bit.mapped_group && bit.mapped_name) {
+                            this.OutputFieldLookup.set([bit.mapped_group, bit.mapped_name].toString(), bit);
```

Comment:
maps support arbitrary types as keys, imo you can omit the string conversion, or is there a specific reason for it? 

```suggestion
                        if (bit.mapped_group && bit.mapped_name) {
                            this.OutputFieldLookup.set([bit.mapped_group, bit.mapped_name], bit);
```

Reply:
Somehow I need to concatenate this two strings, into one single key. Do you have a better alternative in mind?

Reply:
throwing them both into an array while omitting the `.toString` should work, I think?

Reply:
I just tried this, the script loads without syntax failure, but I got always undefined, if I read from the map with this type of key.

Reply:
Ah, thats a shame. Seems like this is common. https://stackoverflow.com/questions/32660188/using-array-objects-as-key-for-es6-map

A workaround with a possible performance advantage would be to use `Symbol.for([a,b])` instead of `[a,b].toString()` but at this point, I'd have to measure whether there is any improvement.